src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discount3To6Days = 20;
+  const dailyRentalCost = 40;
+  const discount7OrMoreDays = 50;
+
+  if (days >= 3 && days < 7) {
+    return days * dailyRentalCost - discount3To6Days;
+  }
```

Comment:
if you start checking for `if days >= 7` - you won't need to checks in this `if`